apps/commudle-admin/src/app/feature-modules/public-community/components/events/events.component.ts

```diff
@@ -19,6 +20,13 @@ export class EventsComponent implements OnInit {
   events: IEvent[] = [];
   eventLoader = false;
 
+  upcomingEvents = [];
+  pastEvents = [];
+  faMapPin = faMapPin;
+
+  width = '390px';
+  height = '230px';
```

Comment:
why er are declare this here, and we are using thins

Reply:
Applied directly in html